src/types/types.ts

```diff
@@ -0,0 +1,21 @@
+export type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+export type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User;
+};
```

Comment:
```suggestion
export type Todo = ServerTodo & {
  user: User;
};
```

Reply:
Good advice didn't know that trick. It makes code better.